topic-operator/src/main/java/io/strimzi/operator/topic/KafkaImpl.java

```diff
@@ -54,7 +53,7 @@ public KafkaImpl(Admin adminClient, Vertx vertx) {
     @Override
     public Future<Void> deleteTopic(TopicName topicName) {
         Promise<Void> handler = Promise.promise();
-        LOGGER.debug("Deleting topic {}", topicName);
+        LOGGER.debugOp("Deleting topic {}", topicName);
```

Comment:
Why does this not belong to any reconciliation? Sounds like deleting or creating topics should always belong to a particular reconciliation.  Applies to most logging done here.